examples/Billing/AddBillingSetup.cs

```diff
@@ -43,9 +45,12 @@ public static void Main(string[] args)
             // The Google Ads customer ID for which the call is made.
             long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
 
-            // Payments setup identifiers. Provide an existing payments account ID to link to the
-            // new billing setup. Must be formatted as "1234-5678-9012-3456".
-            string paymentsAccountId = "INSERT_PAYMENTS_ACCOUNT_HERE";
+            // Either a payment accounts ID or a payment profile ID must be provided for the example
```

Comment:
"payments account ID or a payments profile ID"

There are other occurrences of this issue in the file, please make sure it is consistent.

Reply:
Fixed.